qa/db_assertions.py

```
@@ -0,0 +1,301 @@
+import asyncio
+import os
+import sys
+from datetime import datetime
+from typing import Dict, List, Any
+
+# Add project root to path
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+from app.core.database import async_session
+from sqlalchemy import text
+import asyncpg
```

Ruff (0.11.9)

`asyncpg` imported but unused

Remove unused import: `asyncpg`

(F401)